docs/api-guide.md

```diff
@@ -0,0 +1,26 @@
+# Mod Consortia API Guide
+
+## Tenants API
+
+### Endpoints
+
+| METHOD | URL      | DESCRIPTION                                       |
+|--------|----------|---------------------------------------------------|
+| GET    | /tenants | Return list of tenants based on `limit`, `offset` |
+
+## User Tenants Associations API
+
+### Endpoints
+
+| METHOD | URL                           | DESCRIPTION                                                                  |
+|--------|-------------------------------|------------------------------------------------------------------------------|
+| GET    | /user-tenants                 | Return list of user tenant associations based on `param` , `limit`, `offset` |
```

Comment:
Is this really "param" here, or "query" ? (other modules use the query parameter)

Reply:
in our case, we are not using CQL query for now. because CQL query criteria is not implemented yet.

Reply:
Ok, I will fix it then, Thanks